package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@madie/madie-models",
-  "version": "1.2.1",
+  "version": "1.2.2",
```

Comment:
Should the PR title also be Release 1.2.2 ?

Reply:
yes